### ISSUE TITLE
Rails 5 log_subscriber error

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss/libsys-webforms#616]
I'm seeing this in the rails log on the server and in local development whenever I use OCI-8 to query our Oracle database:
```
Could not log "sql.active_record" event. TypeError: wrong argument type NilClass (must respond to :each) ["/opt/app/libsys/libsys-
webforms/shared/bundle/ruby/2.6.0/gems/activerecord-5.0.7.2/lib/active_record/log_subscriber.rb:37:in `zip'"
<snip>
```

**Comment by jgreben** [comment]
I think this should be fixed when we upgrade to Rails 5.2. See https://github.com/rails/rails/commit/1345511081a7b4562281cf77d9e5aeeae2d3eb02

**Comment by jgreben** [comment]
Fixed by https://github.com/sul-dlss/libsys-webforms/pull/610/commits/0bf099c7ae9135e69e2a5414f9adfd657c5083c9 in #610
